src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzerTests.GuardedCallsTests.cs

```diff
@@ -1267,6 +1267,8 @@ public void ProgramMethod()
 
 [*]
 build_property.TargetFramework = net5
+build_property.TargetFrameworkIdentifier = .NETCoreApp
+build_property.TargetFrameworkVersion = v5.0
```

Comment:
Since these tests aren't actually going through msbuild, it means the props/targets that would normally automatically provide these don't run; hence the tests need to pass it along explicitly.

In all cases I've checked what msbuild proper would generate for a given `TargetFramework` value and ensured the `TargetFrameworkIdentifier` and `TargetFrameworkVersion` values we pass along match that behavior